packages/components/root/src/root.css

```diff
@@ -78,9 +76,7 @@
 }
 :host(.ts-has-sidebar-inner-left.ts-has-sidebar-inner-right) > .content {
 	-ms-grid-columns: var(--ts-aside) minmax(var(--ts-aside), 1fr) var(--ts-aside);
-	grid-template-columns: var(--ts-aside) minmax(var(--ts-aside), 1fr) var(
-			--ts-aside
-		);
+	grid-template-columns: var(--ts-aside) minmax(var(--ts-aside), 1fr) var(--ts-aside);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "grid-template-columns" property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aexperimental-property-usage)